pkg/config/config.go

```diff
@@ -379,6 +379,7 @@ type Config interface {
 	GetBaseConfig() *BaseConfig
 	StorageConfig() StorageConfig
 	AuthConfig() *Auth
+	CustomizationConfig() *CustomizationConfig
```

Comment:
Returning a pointer to a map is quite un-Go-like.  It usually does _not_ do what you would expect.  For instance `nil` means something quite odd here.

This is really a symptom of placing a map as an object, rather than in a top-level field of the object.